srcs/layouts/latn_qwerty_apl.xml

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="utf-8"?>
+<keyboard name="QWERTY (APL)" script="latin" embedded_number_row="true">
+  <row height="0.75">
+    <key c="1" nw="⌶"    ne="¨"               se="!"/>
+    <key c="2" nw="¯"    ne="⍫"               se="@" sw="`"/>
+    <key c="3" nw="&lt;" ne="⍒"               se="#" sw="~"/>
+    <key c="4" nw="≤"    ne="⍋"               se="$"/>
+    <key c="5" nw="="    ne="⌽"               se="%"/>
+    <key c="6" nw="≥"    ne="⍉"               sw="^"/>
+    <key c="7" nw=">"    ne="⊖"               sw="&amp;" se="|"/>
+    <key c="8" nw="≠"    ne="⍟" se="∞"        sw="*"/>
+    <key c="9" nw="∨"    ne="⍱"               sw="("/>
+    <key c="0" nw="∧"    ne="⍲"               sw=")"/>
+  </row>
+  <row>
+    <key c="q" nw="?" ne="⍰"                  se="loc esc"/>
+    <key c="w" nw="⍵" ne="⍹"                  sw="'" se="&quot;"/>
+    <key c="e" nw="∊" ne="⍷"/>
+    <key c="r" nw="⍴"        sw="←" se="⇐"/>
+    <key c="t" nw="⍨"               se="→"/>
+    <key c="y" nw="↑" ne="⌹"        se="÷"    sw="-"/>
+    <key c="u" nw="↓" ne="⍮"        se="×"    sw="+"/>
+    <key c="i" nw="⍳" ne="⍸"                  sw="[" se="{"/>
+    <key c="o" nw="○" ne="⍥"                  sw="]" se="}"/>
+    <key c="p" nw="⍣" ne="⍬" sw="⍞" se="⍡"/>
+  </row>
+  <row>
+    <key c="a" nw="⍺" ne="⍶"                  sw="loc tab" shift="0.25"/>
+    <key c="s" nw="⌈" ne="ᑈ" se="«"/>
+    <key c="d" nw="⌊" ne="ᐵ" se="»"/>
+    <key c="f"  s="…"/>
+    <key c="g" nw="∇" ne="⍢"                  s="_"/>
+    <key c="h" nw="∆" ne="⍙"/>
+    <key c="j" nw="∘" ne="⍤" w="⊣" e="⊢"/>
+    <key c="k"        ne="⌸"/>
+    <key c="l" nw="⎕" ne="⌷" se="⍪"/>
+    <key c=";"  w="⌺"  n="⋄"                  s="," width="0.75"/>
```

Comment:
The symbol at the west position overlaps with the symbol at the center. Why not make the layout a grid ?

Reply:
putting `⌺` on fn+`⋄` works out well as an implicit fix for the west issue.

I assume gridifying would include keeping the shift key 1-wide as-is instead of 1.5 as in the other suggestion? I guess that could work fine (max 0.5 key-width offset from your basic layouts, and 0.25 compared to traditional staggered).

Or maybe I could try harder getting rid of the semicolon char.. (semicolon itself isn't particularly important in APL nor BQN, but housing `⋄` on it is very nice, and generally gives much-needed space for glyphs)

Reply:
> I assume gridifying would include keeping the shift key 1-wide as-is instead of 1.5 as in the other suggestion? I guess that could work fine (max 0.5 key-width offset from your basic layouts, and 0.25 compared to traditional staggered).

I didn't consider the third row when I said grid but if a very frequent symbol could be promoted to its own key, why not :)